include/client/Client.h

```diff
@@ -20,6 +20,8 @@
 #include "rapidjson/stringbuffer.h"
 #include "HTTPRequests.h"
 
+#define LABEL_NAME_MAX_LEN 255
```

Comment:
It's better to avoid the preprocessor for defining simple constants (that aren't needed for preprocessor conditions); this can just be a `const unsigned int`. Furthermore, if it is used only inside `Client::verifyNamespaceName`, it would make the most sense to define it there.

Reply:
Updated [here]{https://github.com/slateci/slate-client-server/pull/171/commits/9bc330263f81e3e022c8cedaa835027099ae0449#diff-e06cb3ee018bdf21480a0079e3248c0c0a11d89c57ad4db86f7d8a847e23a580R3626}